tests/load_testing/README.md

```diff
@@ -0,0 +1,36 @@
+# Refunds Load Testing
+
+Using [Locust](https://github.com/locustio/locust) and [Real Browser Locust](https://github.com/nickboucart/realbrowserlocusts), we can execute a load test on the public front end.
+
+We use [Faker](https://github.com/joke2k/faker) and some random number generation to create unique claims for each time locust runs a complete journey of the claim a refund public front end.
+
+The load test script will refuse to run against protected environments, checking the target url.
+
+### Install
+
+Install python dependencies using pip
+
+``` bash
+pip install -r requirements.txt
```

Comment:
Is it worth containerising this? Would ease python environment managing.

Reply:
A container for this has been hard to get working. 
Using available images, I can't meet the dependencies for locust, realbrowserlocust and firefox.
The solution might be to make a new build process that does it all.
I'll raise this with the team at next planning and we can talk about the value of being able to run this in a container.